src/utils/application.data.ts

```diff
@@ -97,3 +98,17 @@ const getIndexInApplicationData = (req: Request, appData: ApplicationData, key:
     throw createAndLogErrorRequest(req, `application.data getIndexInApplicationData - unable to find object in session data for key ${key} and ID ${id}`);
   }
 };
+
+export const mapOverseasEntityToDTO = (data: any): ICompanyDetails => {
```

Comment:
Need to revisit this "map model" to fit in with the full display of an OE. Please see what's needed for the display / edit full Entity details story.